backend/tests/whereaboutsMaintenanceMode.test.ts

```diff
@@ -0,0 +1,122 @@
+import supertest from 'supertest'
+import express from 'express'
+import nunjucksSetup from '../utils/nunjucksSetup'
+import routes from '../routes'
+
+describe('Whereabouts maintenance mode flag is true', () => {
```

Comment:
Would have liked to also write a Cypress test for this but because the flag is set using `process.env.WHEREABOUTS_MAINTENANCE_MODE` I couldn't think of an easy way to toggle it. Open to ideas!

There's good Cypress test coverage of whereabouts and appointmentDetails and they still pass which is comforting

Reply:
yes, it's a pain. Having some kind of feature toggle system that could modify stuff at runtime would allow testing this. 

In projects I've worked with we generally move tests over to test the new feature and leave the old feature untested - which isn't ideal and doesn't make sense in this case